Workflow does not properly close the issue

https://github.com/LizardByte/community/blob/e016da660ffaf0500f7545fd8c941596f58ac9ae/.github/workflows/rss-notifier.yml#L121

```
 failed to run git: fatal: not a git repository (or any of the parent directories): .git

Error: Process completed with exit code 1.
```

--- a/.github/workflows/rss-notifier.yml
+++ b/.github/workflows/rss-notifier.yml
@@ -142,5 +142,10 @@ jobs:
           close_reason="completed"
           lock_reason="resolved"
 
-          gh issue close ${{ github.event.issue.number }} --comment "${comment}" --reason "${close_reason}"
-          gh issue lock ${{ github.event.issue.number }} --reason "${lock_reason}"
+          gh issue close ${{ github.event.issue.number }} \
+            --comment "${comment}" \
+            --reason "${close_reason}" \
+            --repo "${{ github.repository }}"
+          gh issue lock ${{ github.event.issue.number }} \
+            --reason "${lock_reason}" \
+            --repo "${{ github.repository }}"

